src/components/evaluator/services/znkEvaluatorSrv.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+(function (angular) {
+    angular.module('znk.infra.evaluator').provider('ZnkEvaluatorSrv', function () {
+
+        var _evaluateQuestionGetter;
+
+        this.shouldEvaluateQuestion = function(evaluateQuestionGetter) {
```

Comment:
What does shouldEvaluateQuestion function do ?
What should i provide as evaluateQuestionGetter paramter ?